gatling-jms/src/main/scala/io/gatling/jms/check/JmsBodySubstringCheckMaterializer.scala

```diff
@@ -18,13 +18,13 @@ package io.gatling.jms.check
 
 import io.gatling.core.check.substring.SubstringCheckType
 import io.gatling.core.check.{ CheckMaterializer, Preparer }
+import io.gatling.core.config.GatlingConfiguration
 import io.gatling.jms.JmsCheck
 import javax.jms.Message
 
 object JmsBodySubstringCheckMaterializer {
-  def apply(): JmsBodySubstringCheckMaterializer = new JmsBodySubstringCheckMaterializer()
-}
-
-class JmsBodySubstringCheckMaterializer extends CheckMaterializer[SubstringCheckType, JmsCheck, Message, String](identity) {
-  override protected def preparer: Preparer[Message, String] = JmsMessageBodyPreparers.JmsStringBodyPreparer
+  def apply()(implicit config: GatlingConfiguration): CheckMaterializer[SubstringCheckType, JmsCheck, Message, String] =
```

Comment:
Could you please pass the config explicitly here? We try to limit implicits to the DSL layer.